CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 [//]: # (current developments)
 
+## 24.3.0 (2024-03-12)
+
+### Enhancements
+
+* Show first few characters of undecodeable response if `repodata.json` raises
+  `JSONDecodeError`. (#11804)
+* Update `conda.gateways.subprocess.subprocess_call` to use `text=True` to avoid manual encoding/decoding. (#13240)
+* Adds a new plugin hook giving plugin authors the ability to define new settings. (#13554)
+* Optimize module imports to speed up `conda activate`. (#13567 via 13568)
+* Move `conda env export` to `conda export` and alias the old command to the new command. (#13577)
+* Report progress while running `conda install --revision <idx>`. (#13611)
+* Add `conda.testing.tmp_channel` pytest fixture to create a temporary local channel for testing. (#13634)
+
+### Bug fixes
+
+* Print traceback on `KeyboardInterrupt` instead of raising another
+  `AttributeError` exception, when conda debugging logs are enabled. (#13531)
```

Comment:
```suggestion
* Print traceback on `KeyboardInterrupt` instead of raising another `AttributeError` exception, when conda debugging logs are enabled. (#13531)
```